pennylane/_grad.py

```diff
@@ -181,3 +181,75 @@ def _jacobian_function(*args, **kwargs):
         return _np.stack([_jacobian(func, arg)(*args, **kwargs) for arg in argnum]).T
 
     return _jacobian_function
+
+
+def finite_diff(F, x, i=None, delta=0.01):
```

Comment:
Is 0.01 the optimal value of delta to evaluate the derivative of molecular Hamiltonians with respect to coordinates in a.u.? If not, it might be good to have that value as default.

Reply:
0.01 a. u. is the default you find in qchem packages. That is the motivation to keep it as a default. But in general, finite differences are susceptible to to this value. For the Hamiltonian of small molecules, 0.01 seems to be OK.

Reply:
For example, it compares well with the HF analytical gradients:

analytical gradients (PySCF):
```
H  0.,  0.,  0.01374296
H  0.,  0., -0.01374296
```

ours:
```
H  0.,  0.,  0.01375118
H  0.,  0., -0.01375118
```
of course, we can check for larger systems e.g., water.

Reply:
0.01 seems like a very reasonable default. Remember the user can always change it to any other desired value